### ISSUE TITLE
Service account from a non-GCE VM

### ISSUE THREAD

**Issue by jean-philippe-martin** [broadinstitute/gatk#2425]
@kcibul asks: how do I run using a service account from a non-GCE VM

**Comment by lbergelson** [comment]
@jean-philippe-martin Travis runs the cloud tests using a service account on a non gcs machine.  (at least I assume it's not a gcs vm, I think they use amazon cloud although that could have changed...)  All we do to log in is:

```
gcloud config set project broad-dsde-dev;
gcloud auth activate-service-account --key-file servicekey.json
```

@kcibul Where you connecting from the broad network?  I've had problems connecting to gcs from home because of IP  restrictions on the broad projects.  Maybe your gsutil has some configuration setup to do tunneling but gatk doesn't?  Sort of a long shot since I would expect both to not work if either doesn't.

**Comment by jean-philippe-martin** [comment]
@lbergson's instructions above are good, but somehow they did not work for me. I was able to follow the [application default credentials](https://developers.google.com/identity/protocols/application-default-credentials) instructions, though. Here are the steps I took:

 1. create a new service account on the Google Cloud web page and download the JSON key file.
 2. gcloud auth activate-service-account --key-file "$PATH_TO_THE_KEY_FILE"
 3. export GOOGLE_APPLICATION_CREDENTIALS="$PATH_TO_THE_KEY_FILE"

I cleared my credentials first to make sure that the access worked because of the above steps, not because of other credentials.

After those steps, gatk was able to run from my desktop and access files using the service account credentials. `gsutil ls` worked as well.
